package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/core-monorepo",
-  "version": "167.0.0",
+  "version": "168.0.0",
```

Comment:
Whoops, I forgot to update this (was lost when I was reverting files).
Here was the original PR: https://github.com/MetaMask/core/pull/4469